exercise/api/csv/aggregate_points.py

```diff
@@ -56,8 +56,11 @@ def aggregate_points(request, profiles, taggings, exercises, aggregate, number):
             ('UserID', uid),
             ('Email', profile.user.email),
             ('StudentID', profile.student_id),
+            ('FirstName', profile.user.first_name),
+            ('LastName', profile.user.last_name),
             ('Name', profile.user.first_name + ' ' + profile.user.last_name),
             ('Tags', '|'.join(user_tags)),
+            ('Organization', profile.organization),
```

Comment:
What is displayed if the profile doesn't have an organization listed? (I'm not sure if an organization is listed if the user has logged in via social authentication.)

Reply:
All right, good. I just wanted to make sure that it's not "Null" or even worse, break the table.